Sprint-1/destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,26 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+
+
+// Task 1:
+
+console.log("Gryffindor Members:");
+
+hogwarts.forEach(({ firstName, lastName, house }) => {
```

Comment:
This works really well! 👏
Something you can do (only a suggestion) is to use `.filter` to only keep the records where house === "Gryffindor", before `.forEach`.
This will make your code more concise.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

_(The same could be done to filter and only keep teachers who have pets in task 2 below)_